yarn.lock

```diff
@@ -18243,7 +18243,7 @@ __metadata:
     yargs: "npm:^17.6.2"
   bin:
     openmrs: ./dist/cli.js
-  checksum: 10/7d4b13b33cdfabbc3e0362682aeb81a71a770133d81bc35ad232e6d0e0bc7e5f665fbc5472020c7bbd661c01a6aa342cd1012a64b0970dc2c7410c8a87069c14
+  checksum: 10/4579c3ec89aff07d47da479c6ec088acff96c735b130e8335e3a0f9244412927ec1eaada9b94d6181841381cbeb8fb5d49552f259ac0b20a76d1b0b3581882c4
```

Comment:
i would expect this ticket to only involve scss classes only basing on the ticket title.

Reply:
Yes and I did only edit the scss file but some changes were made when I updated framework and also pulled latest changes from upstream
@sherrif10